packages/fiori/src/IllustratedMessage.js

```diff
@@ -327,6 +356,10 @@ class IllustratedMessage extends UI5Element {
 	}
 
 	handleResize() {
+		if (this.size !== IllustrationMessageSize.Auto) {
```

Comment:
In theory we can even omit the resize handler, because it's still active and executed on every resize, although it won't change anything. But it's not that heavy, so I am ok to stay like this.

Reply:
I see _handleResize is currently registered in onEnterDOM and for the purpose this may have to be done in onBeforeRendering as the property may be changed. Decided to merge like this, might do a small optimisation in a separate PR.